systems/plants/test/testURDFmex.m

```diff
@@ -14,76 +14,117 @@
   error('Drake:MissingDependency','testURDFmex requires that urdfManipulatorDynamicsTest is built (from the command line).  skipping this test');
 end
 
-tol = 1e-2; % low tolerance because i'm writing finite precision strings to and from the ascii terminal
+tol = .1; % low tolerance because i'm writing finite precision strings to and from the ascii terminal
 
-for urdf = allURDFs()'
+if nargin<1 || isempty(urdfs)
+  urdfs = allURDFs();
+elseif ~iscell(urdfs)
+  urdfs = {urdfs};
+end
+
+for urdf = urdfs'
   urdffile = GetFullPath(urdf{1});
   fprintf(1,'testing %s\n', urdffile);
-  r = RigidBodyManipulator(urdffile,struct('floating',true,'use_new_kinsol',true));
-
+  w = warning('off','Drake:RigidBodyManipulator:UnsupportedVelocityLimits');
+  warning('off','Drake:RigidBodyManipulator:UnsupportedContactPoints');
+  warning('off','Drake:RigidBodyManipulator:ReplacedCylinder');
+  try 
+    r = RigidBodyManipulator(urdffile,struct('floating',true,'use_new_kinsol',true));
+  catch ex
+    if strncmp(ex.identifier,'Drake:MissingDependency:',24)
+      disp(' skipping due to a missing dependency'); 
```

Comment:
What is the magic number 24 here? Could you extract it out into a named variable?